docs/tutorial/updates.md

```diff
@@ -99,6 +99,21 @@ it will receive an update for each new
 publish.
 
 ## Applying Updates
+1> Autoupdater
+An autoUpdater is an automated deployment system that regularly updates your users with the latest version of your application. The advantages of using an auto updater include:
+
+Automating the update of your user applications,
+Removing the need to download a new setup file for each update
+
+2> why should we use autoupdater & its advantages:
+The main pro of keeping your automatic update functions enabled is that any newly-discovered security or performance issues are, well, automatically updated.
```

Comment:
This is copied from https://www.ntiva.com/blog/the-pros-and-cons-of-automatic-software-updates.